### DISCUSSION TITLE
Trigger hover state on parent hover

### DISCUSSION THREAD

**Discussion by aasem-a** [elementor/elementor#21355] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I find it odd when some very basic Gutenberg blocks plugins allow me to assign hover colors/effects based on the parent or child hover!
We need to have a switch to allow the haver effect of the child element to be triggered on parent hover.
I've found other requests for similar features but for unrelated classes. This is for the same widget and its child elements.
Please consider.
![screenshot-2023 02 15-14_18_37](https://user-images.githubusercontent.com/125462862/219027768-7647e8f2-9e58-470a-82b4-82f107a65060.png)

### Describe the solution you'd like

A switch to turn on the haver effect of the child element on parent hover

### Describe alternatives you've considered

This CSS does the trick but it applies to all similar widgets on the website! Adding custom class ids didn't work!
.elementor-widget-icon-box .elementor-icon-box-wrapper:hover > .elementor-icon-box-icon * {
	color: blue !important;
}

**Comment by tcvailoces** [comment]
+1 for this, we need this feature to be included on the next updates allowing us to change state to child elements when hovering the parent container because as of now we are using custom css

**Comment by kimu** [comment]
+1 for this feature, extended to also transform, backgrounds and other part where there is a "hover" option

**Comment by khashayarghajar** [comment]
+1

**Comment by rutgerrrrr** [comment]
+1
